### PR TITLE
add/edit documentation and instructions

### PR DESCRIPTION
**Pull request by erinchoi2001**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,8 +16,93 @@ <h1>The Revamped Recessionary Ranch (feat. The Broke Farmer)</h1>
...
+                Press P to toggle the pop-up window that shows you what seeds you can plant and how many seeds you have for each ingredient. Click the image of the ingredient you want to plant.
+              </li>
+              <li>
+                Walk up to a plot of soil and press SPACE to plant a seed. Wait while the plant goes through 4 stages of growth - when it's fully matured and looks as it appears in the inventory or seed window, press SPACE in front of it to harvest it.
+              </li>
+              <li>
+                You can speed up the growth of your crops by watering them. Walk up to the pond and press SPACE to fill your watering can. Walk up to a plant and press SPACE to water it.
+              </li>
+              <li>
+                Plants can be watered as many times as you want until they're fully grown. However, your water is limited - you can only water plants 5 times before before the watering can is empty again!
```
Comment (tangym27) [inline index.html]: "before before"

File: index.html [hunk 2]
```diff
@@ -17,8 +16,93 @@ <h1>The Revamped Recessionary Ranch (feat. The Broke Farmer)</h1>
...
+              </li>
+            </ul>
+          </div>
+
+          <div>
+            <h3>Udderly Amoosing Cow Game</h3>
+            <p>You can only obtain milk through this VR minigame!</p>
+            <ul>
+              <li>
+                Walk up to the cow surrounded by flower trees inside the fenced pen at the top right of the farm. Press SPACE next to the cow to begin the VR minigame. You'll be transported to a 3D world!
```
Comment (tangym27) [inline index.html]: can you add holy to "cow"
so 
"holy cow" 
hehehe

File: index.html [hunk 3]
```diff
@@ -17,8 +16,93 @@ <h1>The Revamped Recessionary Ranch (feat. The Broke Farmer)</h1>
...
+              </li>
+              <li>
+                Use your mouse to rotate your view, and click on and collect the drops of milk falling from the sky. 5 drops makes 1 bottle of milk! You can see how much milk you've collected on the display board next to the cow.
+              </li>
+              <li>
+                Collect 10 milk bottles' worth of milk drops to end the game automatically and return to the farm. If you want to end the game early, press on the yellow box just below the display board.
```
Comment (tangym27) [inline index.html:78]: don't think the game ends automatically at ten?

**Review by tangym27 — COMMENTED** [review]
vibes

File: index.html (hunk 3 above)
Comment (erinchoi2001) [reply index.html:78]: it should trigger the same thing as clicking the yellow button when you collect 10!

**Pull request merged**